src/main/java/sc/fiji/bdvpg/bdv/navigate/ViewerTransformChanger.java

```diff
@@ -38,31 +39,53 @@
  *
  * See ViewTransformSetAndLogDemo for a usage example
```

Comment:
Explain in the header why there's an animation duration, and how setting a negative (or 0?) value leads to instant displacement.

Reply:
ok!